dobi.yaml

```diff
@@ -2,17 +2,18 @@
 # alias
 # ===================================================
 
-alias=bb-buildingblock-build:
-  tasks: []
+alias=bb-gitversion-build:
+  tasks:
+  - gitversion-build
   annotations:
-    description: "-> build all content (not implemented yet)"
+    description: "-> build all content"
 
-alias=bb-buildingblock-deploy:
+alias=bb-gitversion-deploy:
```

Comment:
deadbeef? it there something we deploy in this context?

Reply:
i.e. future push to docker hub.

Reply:
i resolve ... but i would expect this alias to be in the future commit which dies the deploy